src/main/resources/themes/palenight-theme.json

```diff
@@ -201,7 +201,7 @@
       "hoverColor": "#444267",
       "hoverMaskColor": "#444267",
       "inactiveMaskColor": "#292D3E",
-      "inactiveColoredFileBackground": "#292D3E2",
+      "inactiveColoredFileBackground": "#292D3E",
```

Comment:
What happened because of this change?

Reply:
Odd, iirc that last number should control the opacity of the colour.

Reply:
#RRGGBBAA or #RGBA. Maybe it should be #292D3E02. I'll check now...

Reply:
You might be right, its been a while... But worth checking at least.

Reply:
I found!
>  "inactiveColoredFileBackground": "#00000000"

https://plugins.jetbrains.com/docs/intellij/themes-customize.html#customizing-the-borders-of-ui-controls